### PR TITLE
KM94 feat(declarativecluster) PART 4: Add no user input modifications 

### PR DESCRIPTION
**Pull request by deifyed**
## Description

This PR contains the necessary changes I had to do to existing code to disable user input. Mainly the tactic here was to duplicate the existing functionality and remove Ask references.

### CODE REVIEW

File: pkg/config/load/user.go [hunk 1]
```diff
@@ -50,6 +50,37 @@ all the time, if you so choose.
...
+		}
+
+		userDataPath, err := c.GetUserDataPath()
+		if err != nil {
+			return fmt.Errorf("getting user data path: %w", err)
+		}
+
+		data := state.NewUser()
+
+		data.User.Username = "ooo306134" // TODO: from env var?
```
Comment (ivaruf) [inline pkg/config/load/user.go:67]: Hard coded user?

File: pkg/okctl/okctl.go [hunk 2]
```diff
@@ -746,6 +746,12 @@ https://www.passwordstore.org/
 		_ = k.Store(keyring.KeyTypeUserPassword, password)
 	}
 
+	//machineAuth := aws.NewAuthMachine( // TODO: make AuthMachine work and add it to 772:aws.New
```
Comment (ivaruf) [inline pkg/okctl/okctl.go:749]: Commented code?

**Review by ivaruf — CHANGES_REQUESTED** [review]
Just a few small issues, see comments.

File: pkg/okctl/okctl.go (hunk 2 above)
Comment (deifyed) [reply pkg/okctl/okctl.go:749]: Hmm. Woops. This is code from the auth part which should not be a part of this PR. I'll remove

File: pkg/config/load/user.go (hunk 1 above)
Comment (deifyed) [reply pkg/config/load/user.go:67]: :D good catch

File: pkg/credentials/aws/aws.go [hunk 3]
```diff
@@ -209,6 +209,69 @@ func DefaultStsProvider(sess *session.Session) stsiface.STSAPI {
 // to missing or expired credentials
 type PopulateFn func(*AuthSAML) error
 
+//type AuthMachine struct {
```
Comment (yngvark) [inline pkg/credentials/aws/aws.go:212]: Perhaps put in another branch instead of letting these comments go to master?

**Review by yngvark — CHANGES_REQUESTED** [review]

File: pkg/credentials/aws/aws.go (hunk 3 above)
Comment (deifyed) [reply pkg/credentials/aws/aws.go]: Yeah, I've removed them from this branch

**Review by yngvark — APPROVED** [review]

**Pull request merged**